providers/os/resources/packages/windows_packages.go

```diff
@@ -601,3 +614,41 @@ func (win *WinPkgManager) Files(name string, version string, arch string) ([]Fil
 	// not yet implemented
 	return nil, nil
 }
+
+// findMsSqlHotfixes returns a list of hotfixes that are related to Microsoft SQL Server
+// The list is sorted by the hotfix id
+func findMsSqlHotfixes(packages []Package) []Package {
+	sqlHotfixRegExp := regexp.MustCompile(`^Hotfix .+ SQL Server`)
```

Comment:
nit: why don't define this as a global var. In that way we won't need to compile the regex every time the function gets called. We have done this elsewhere too

Reply:
Fixed.